great_expectations/expectations/core/expect_column_pair_values_to_be_in_set.py

```diff
@@ -13,55 +13,22 @@ class ExpectColumnPairValuesToBeInSet(ColumnPairMapExpectation):
     expect_column_pair_values_to_be_in_set is a \
     [Column Pair Map Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations).
 
-    For example:
-    ::
-        >>> d = {'fruit': ['appple','apple','apple','banana','banana'],
-                'color': ['red','green','yellow','yellow','red']}
-        >>> my_df = pd.DataFrame(data=d)
-        >>> my_df.expect_column_pair_values_to_be_in_set(
-                'fruit',
-                'color',
-                [
-                    ('apple','red'),
-                    ('apple','green'),
-                    ('apple','yellow'),
-                    ('banana','yellow'),
-                ]
-        )
-        {
-            "success": false,
-            "meta": {},
-            "exception_info": {
-                "raised_exception": false,
-                "exception_traceback": null,
-                "exception_message": null
-            },
-            "result": {
-                "element_count": 5,
-                "unexpected_count": 1,
-                "unexpected_percent": 20.0,
-                "partial_unexpected_list": [
-                    [
-                        "banana",
-                        "red"
-                    ]
-                ],
-                "missing_count": 0,
-                "missing_percent": 0.0,
-                "unexpected_percent_total": 20.0,
-                "unexpected_percent_nonmissing": 20.0
-            }
-        }
+    Column Pair Map Expectations are evaluated for a pair of columns and ask a yes/no question about the row-wise relationship between those two columns.
```

Comment:
See previous edit recommendations and update all similar text so it is consistent.